API_Database.py

```diff
@@ -9,29 +10,31 @@ def insert(date):
     #c.execute("CREATE DATABASE API")
     
     #Create The DB
-    c.execute("CREATE TABLE IF NOT EXISTS API_Results (Date TEXT PRIMARY KEY, Movie_Title TEXT, NASA_URL TEXT, WIKI_URL TEXT)")
+    table_sql = "CREATE TABLE IF NOT EXISTS API_Results (Date TEXT PRIMARY KEY, Movie_Title TEXT, Movie_Url TEXT, Nasa_Title, Nasa_Url TEXT, Wiki_Url TEXT)"
```

Comment:
Can the database get created before the insert function runs?

Reply:
It would probably be safest to create the database when the user visits the home page. That way I could put a check on the bookmark button to see if there is any rows in the database. If the user has not created the database before pressing the bookmark button I imagine the app will crash.

Reply:
Or create the DB when the app loads. Add a create_table function in your DB module, call it when you start the app.